integration-tests/tests/connection_set_up.py

```diff
@@ -0,0 +1,102 @@
+import time
+import requests
+import os
+import logging
+
+import signal
+
+logging.basicConfig(level=logging.WARNING)
+
+
+def delete_old_database():
+    try:
+        os.remove("../example_project/example_project/db.sqlite3")
+        logging.debug("Database file in example_project DELETED!")
+    except OSError:
+        logging.debug("No database file found.")
+        pass
+
+
+def obtain_csrftoken(session):
+    """
+    A CSRF cookie token is required in order to not get a 403 Forbidden response by
+    the post to the inputs.
+    :return: String representing the token.
+    """
+
+    return session.cookies['csrftoken']
+
+
+def create_session():
+    """
+    A integration test utility to create a browser session request for a single test.
+    :return: A session object for requests.
+    """
+
+    return requests.Session()
+
+
+def send_get_request(session, url):
+    """
+    Attempts to send a GET to the url. Server is already up as the test should
+    be calling the `is_server_healthy()` function before calling this.
+
+    :param session: Object representing the browser session.
+    :param url: String, containing 'http://', with the URL of the target URL.
+    :return: Response of GET.
+    """
+    response = None
+
+    logging.debug("Attempting a GET request for the provided URL...")
+    try:
+        response = session.get(url)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        print(e)
+
+    return response
+
+
+def send_post_request(session, url, data):
+    """
+    Attempts to send a POST to the url. Server is already up as the test should
+    be calling the `is_server_healthy()` function before calling this.
+
+    :return: Response of POST.
+    """
+    response = None
+
+    logging.debug("Attempting a POST request for the provided URL...")
+
+    try:
+        response = session.post(url, data=data)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as e:
```

Comment:
are we sure that we want to catch this? by the time this is called the server should be up, if it goes down we want the report this?

Reply:
perhaps we can use the `logging` package here

Reply:
This isn't a test so yes, I think we should catch it. Tests will have a HTTP code assertion to see if everything was successful after this POST request is made. 

Instead of failing logic by getting random exceptions, I think we should fail it by getting a different response than expected imo.